app/javascript/src/lib/dayjs.js

```diff
@@ -0,0 +1,6 @@
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+dayjs.extend(relativeTime);
+
+export default dayjs;
```

Comment:
Had you verified whether this extension is working as intended from the UI? Because from my understanding you haven't imported this file anywhere yet and thus this extending wouldn't have any effect.

You should import this file once at the root component which is App.jsx.